src/org/httpkit/client.clj

```diff
@@ -193,7 +195,9 @@
                                 ;; only the 4 support now
                                 (case as :auto 1 :text 2 :stream 3 :byte-array 4))
         effective-proxy-url (if proxy-host (str proxy-host ":" proxy-port) proxy-url)
-        cfg (RequestConfig. method headers body timeout keepalive effective-proxy-url tunnel?)]
+        conn-timeout (if timeout timeout conn-timeout)
```

Comment:
Is there a reason we wouldn't want this to be `(or conn-timeout timeout)` instead? I.e. use the more specific `conn-timeout` when provided, otherwise fall back to general `timeout`?